src/portal/package-lock.json

```diff
@@ -176,13 +176,15 @@
                     "version": "1.40.0",
                     "resolved": "https://registry.npmjs.org/mime-db/-/mime-db-1.40.0.tgz",
                     "integrity": "sha512-jYdeOMPy9vnxEqFRRo6ZvTZ8d9oPb+k18PKoYNYUe2stVEBPPwsln/qWzdbmaIvnhZ9v2P+CuecK+fpUfsV2mA==",
-                    "dev": true
```

Comment:
did you change the package.json? If not , please don't submit this package-lock.json.